recipes/boost-ext-simd/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class BoostExSimdConan(ConanFile):
+    name = "boost-ext-simd"
+    description = ("Portable SIMD computation library - was proposed as a "
+                   "Boost library"
+                   )
+    homepage = "https://github.com/procxx/boost.simd"
+    topics = ("conan", "boost", "simd")
+
+    license = "BSL-1.0"
+    url = "https://github.com/procxx/boost.simd"
+
+    generators = "cmake", "cmake_find_package"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    requires = "boost/[>=1.60]"
+
+    # subfolders to force out-of-source build
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    # cache CMake helper to avoid multiple CMake configurations
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
```

Comment:
Use explicit versions, no version ranges.
Also, put the requirements in a method, not in ``requires` attribute.

This is done in cci for uniformity.

```suggestion
            del self.options.fPIC

    def requirements(self):
        self.requires("boost/1.76.0")

    def source(self):
```